packages/regl-worldview/src/Worldview.js

```diff
@@ -32,7 +33,7 @@ export type BaseProps = {|
   cameraState?: CameraState,
   onCameraStateChange?: (CameraState) => void,
   defaultCameraState?: CameraState,
-
+  mouseClickRadius: number,
```

Comment:
I'm not sure if we should support this as a property or just hardcode it. The fewer properties the better, because each one of these we will have to support indefinitely (until we bump major versions).

Reply:
(And I don't see any good reason for people to change this property?)

Reply:
You are right. The fewer props the better. Just moved.